<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Battle</class>
 <widget class="QWidget" name="Battle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="pushButton_UpTour">
     <property name="text">
      <string>Upgrade Tournament</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_DuRace">
     <property name="text">
      <string>Duel Race</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_level"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_rarity">
       <item>
        <property name="text">
         <string>Common</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rare</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Epic</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Legendary</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_type">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_infoA">
       <property name="text">
        <string>你的HP:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_HPA">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar_A">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_infoB">
       <property name="text">
        <string>对方的HP:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_HPB">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar_B">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView_chellenges"/>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser_info"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,7">
     <item>
      <widget class="QPushButton" name="pushButton_back">
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Battle">
       <property name="text">
        <string>Battle!</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,1">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_skip">
       <property name="text">
        <string>skip</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
